### PR TITLE
Hidetext: Add 'hidealttext' alias

### PR DESCRIPTION
**Pull request by panpawn**

### CODE REVIEW

File: chat-commands.js [hunk 1]
```diff
@@ -2228,7 +2229,7 @@ exports.commands = {
 	},
 	hidetexthelp: [
 		"/hidetext [username] - Removes a locked or banned user's messages from chat (includes users banned from the room). Requires: % (global only), @ * # & ~",
-		"/hidealtstext OR /hidetextalts [username] - Removes a locked or banned user's messages, and their alternate account's messages from the chat (includes users banned from the room).  Requires: % (global only), @ * # & ~",
+		"/hidealtstext OR /hidealttext OR /hidetextalts [username] - Removes a locked or banned user's messages, and their alternate account's messages from the chat (includes users banned from the room).  Requires: % (global only), @ * # & ~",
```
Comment (Zarel) [inline chat-commands.js]: ...why are there any `OR`s at all here?

We literally only use `OR` for shortcuts, not for all aliases. I'd just leave whichever one makes the most sense, which I'd argue is `/hidealtstext`.

**Comment by panpawn** [comment]
Updated

**Comment by Zarel** [comment]
I'm saying you should change it so there's no `OR` at all

**Comment by Zarel** [comment]
Thanks!

**Pull request merged**